tests/beman/scope/scope.test.cpp

```diff
@@ -0,0 +1,3 @@
+int main() {
```

Comment:
I believe the licence information should also be added.
```suggestion
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

int main() {
```

Reply:
thx for catching this!